src/pages/cli/graphql/override.mdx

```diff
@@ -118,6 +118,74 @@ export function override(resources: AmplifyApiGraphQlResourceStackTemplate) {
 }
 ```
 
+### Example - Configure OpenSearch Streaming function name
+Override the name of the AWS Lambda searchable streaming function
+
+```ts
+import { AmplifyApiGraphQlResourceStackTemplate } from '@aws-amplify/cli-extensibility-helper';
+
+export function override(resources: AmplifyApiGraphQlResourceStackTemplate) {
+  resources.opensearch.OpenSearchStreamingLambdaFunction.FunctionName = 'CustomFunctionName';
+}
+
+```
+
+### Example - Configure OpenSearch instance version
+
+Override the elasticsearchVersion in the OpenSearch domain created by @searchable
+
+```ts
+import { AmplifyApiGraphQlResourceStackTemplate } from '@aws-amplify/cli-extensibility-helper';
+
+export function override(resources: AmplifyApiGraphQlResourceStackTemplate) {
+  resources.opensearch.OpenSearchDomain.elasticsearchVersion = 'OpenSearch_1.3';
+}
+```
+### Example - Configure OpenSearch instance type
+
+Override the type of instance launched into the OpenSearch domain created by @searchable
+
+```ts
+import { AmplifyApiGraphQlResourceStackTemplate } from '@aws-amplify/cli-extensibility-helper';
+
+export function override(resources: AmplifyApiGraphQlResourceStackTemplate) {
+    resources.opensearch.OpenSearchDomain.elasticsearchClusterConfig = {
+        ...resources.opensearch.OpenSearchDomain.elasticsearchClusterConfig,
+        instanceType: "m5.large.search"
+    }
+   
+}
+```
+### Example - Configure OpenSearch instance count
+Override the number of instances launched into the OpenSearch domain created by @searchable
+
+```ts
+import { AmplifyApiGraphQlResourceStackTemplate } from '@aws-amplify/cli-extensibility-helper';
+
+export function override(resources: AmplifyApiGraphQlResourceStackTemplate) {
+    resources.opensearch.OpenSearchDomain.elasticsearchClusterConfig = {
```

Comment:
the tab spacing is different here than in the previous examples where it appears to be using 2 spaces https://github.com/aws-amplify/docs/pull/5291/files#diff-edcb511e6975a1b33dbcf4d4d0f70e70f3465fb841c01016f01988df6d151fb3R141